### DISCUSSION TITLE
posts/Go/prometheus-operators/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#237] [Announcements]
https://www.liwenzhou.com/posts/Go/prometheus-operators/ 

 李文周的 Blog 提供 Go 语言中文教程,李文周博客,Go 语言学习之路是我整理的一个免费教程，包含部分视频教程，也是非常适合新手学习Go语言的入门教程。Prometheus 提供了一种名为 PromQL (Prometheus Query Language) 的功能性查询语言，允许用户实时选择和聚合时间序列数据。表达式的结果既可以显示为图形，也可以在 Prometheus 的表达式浏览器中显示为表格数据，或者被外部系统通过API使用。本文介绍了 Prometheus 中支持的所有运算符及其查询语法。

**Comment by here-tunan** [comment]
非常不错的文章！很有收获～ 谢谢
